### PR TITLE
Add Text Utilities

### PR DESCRIPTION
**Pull request by kavyakunder**
Text Utilities
-headings
-small text
-gray text
-center text

text-utiltiess

### CODE REVIEW

File: Components/Text-Utilities/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.h2-heading {
+  font-size: 2.8rem;
+  font-weight: 600;
+}
+
+.h3-heading {
+  font-size: 2.4rem;
+  padding: ();
```
Comment (Tanay-J) [inline Components/Text-Utilities/text-utilities.css]: Not sure about use of this line. Please check.

**Review by Tanay-J — COMMENTED** [review]
Else the code looks good

**Review by RakshithBhat03 — COMMENTED** [review]
Looks good

File: Components/Text-Utilities/text-utilities.css (hunk 1 above)
Comment (kavyakunder) [reply Components/Text-Utilities/text-utilities.css]: Removed it.

**Pull request merged**